CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
   - `simple_string`
   - `json_value`
   - `json`
+- Added support for method `from_json` in the following type classes:
```

Comment:
we could possible combine all types related improvement summary in a common changelog.

Reply:
something like

```
- Added new methods to enhance data type handling and JSON serialization/deserialization:
  - To `DataType`, its derived classes, and `StructField`:
    - `type_name`: Returns the type name of the data.
    - `simple_string`: Provides a simple string representation of the data.
    - `json_value`: Returns the data as a JSON-compatible value.
    - `json`: Converts the data to a JSON string.
  - To `ArrayType`, `MapType`, `StructField`, and `StructType`:
    - `from_json`: Enables these types to be created from JSON data.
```

Reply:
sure, will do